README.md

```diff
@@ -66,9 +66,9 @@ Created out/Alice.key
 Created out/Alice.csr
 ```
 
-certstrap requires at least one of `-common-name`, `-ip`, or `-domain` flags to be set in order to generate a certificate signing request.  The CN for the certificate will be found from these fields.
+certstrap requires at least one of `-common-name`, `-ip`, `-domain`, or `-uri` flags to be set in order to generate a certificate signing request.  The CN for the certificate will be found from these fields.
```

Comment:
This line isn't quite accurate.  We require at least one of -common-name or -domain now